### PR TITLE
Fix simple XAML errors

### PR DESCRIPTION
**Pull request by piksel**
## What does the pull request do?
Removes duplicate `Content` in Control Catalog sample, fixes 6 errors¹
Wraps `GeometryDrawing` content in `GeometryDrawing.Geometry`, fixes 369 errors¹

¹ As reported by JetBrains Rider upon opening the Avalonia solution.

## What is the current behavior?
Errors are reported for the solution in Rider, although XAML seems to be interpreted correctly on runtime.

## What is the updated/expected behavior with this PR?
Fewer errors to distract from the actual problems when working with the solution.

## Checklist

- ~~Added unit tests (if possible)?~~
- ~~Added XML documentation to any related classes?~~
- ~~Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation~~

## Breaking changes
None

## Obsoletions / Deprecations
None

## Fixed issues
None

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/AvaloniaUI/Avalonia?pullRequest=6020) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/AvaloniaUI/Avalonia?pullRequest=6020) before we can accept your contribution.<br/><br/>:x: piksel [sign now](https://cla.dotnetfoundation.org/AvaloniaUI/Avalonia?pullRequest=6020)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/AvaloniaUI/Avalonia?pullRequest=6020) it.</sub>

File: src/Avalonia.Diagnostics/Diagnostics/Controls/FilterTextBox.axaml [hunk 1]
```diff
@@ -27,7 +27,9 @@
...
+                    <GeometryDrawing.Geometry>
```
Comment (maxkatz6) [inline src/Avalonia.Diagnostics/Diagnostics/Controls/FilterTextBox.axaml:30]: But why it is reported as an error?

Comment (piksel) [reply src/Avalonia.Diagnostics/Diagnostics/Controls/FilterTextBox.axaml:30]: Not sure. It does have the `ContentAttribute` set, so it seems like it should work... Searched the code base for other usages where the Geometry is set using content, but couldn't find any.
This is the error presented in Rider:
```
  <Avalonia.Diagnostics>\Diagnostics\Controls\FilterTextBox.axaml:1247 Text is not expected here
```
and visual studio:
image

Comment (maxkatz6) [reply src/Avalonia.Diagnostics/Diagnostics/Controls/FilterTextBox.axaml:30]: It feels like it should be reported to JetBrains.
Though I am fine with change in this PR anyway.

**Review by maxkatz6 — APPROVED** [review]

File: src/Avalonia.Diagnostics/Diagnostics/Controls/FilterTextBox.axaml (hunk 1 above)
Comment (piksel) [reply src/Avalonia.Diagnostics/Diagnostics/Controls/FilterTextBox.axaml:30]: Maybe... I'm not sure how the (A)XAML-parsing works inside the IDEs. As I mentioned above, this happens in Visual Studio as well. Unlike WPF/UWP errors though, the errors cannot be found inside the Error List / Issue count:
image

**Pull request merged**